peachjam_search/admin.py

```diff
@@ -1,6 +1,11 @@
 from background_task.admin import TaskAdmin
+from django.contrib import admin
 from django.utils.timezone import now
 
+from peachjam_search.models import SearchFeedback
+
+admin.site.register(SearchFeedback)
```

Comment:
Please make all fields readonly.

Add a field for search_trace which links to the `search_trace` url (see `peachjam_search.urls') - ref `DocumentAdmin.work_link` so we can click through to it easily

Reply:
also set `date_hierarchy = 'created_at'`